File: backend/MovieOnline/MovieOnline/Data/Bases/Comment.cs
using System;

namespace MovieOnline.Data.Bases
{
    public class Comment
    {
        public int Id { get; set; }

        public string Content { get; set; }

        public DateTime DateCreated { get; set; }

        public int UserId { get; set; }

        public int? MovieId { get; set; }

        public int? EpisodeId { get; set; }
    }
}

File: frontend/mobile/MovieOnline/app/src/main/java/com/neptune/movieonline/models/Comment.java
package com.neptune.movieonline.models;

import java.util.Date;

/**
 * Created by Neptune on 3/28/2018.
 */

public class Comment {

    private Integer id;
    private String content;
    private Date dateCreated;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public Date getDateCreated() {
        return dateCreated;
    }

    public void setDateCreated(Date dateCreated) {
        this.dateCreated = dateCreated;
    }
}
